src/Microsoft.AspNet.Razor.Runtime/Resources.resx

```diff
@@ -153,4 +153,7 @@
   <data name="TagHelperDescriptorFactory_Tag" xml:space="preserve">
     <value>Tag</value>
   </data>
+  <data name="TagHelperDescriptorFactory_InvalidBoundAttributeName" xml:space="preserve">
+    <value>Invalid tag helper bound property '{0}' binding to HTML attribute '{1}' for type '{2}'. Tag helpers cannot bind to attributes beginning with '{3}'.</value>
```

Comment:
first sentence doesn't read good.  suggest leaving out `binding to HTML attribute '{1}'` and simplifying to `Invalid tag helper bound property '{0}.{1}'.`  `{0}` is the type and `{1}` is the property.